deepchem/metrics/score_function.py

```diff
@@ -229,7 +229,7 @@ def concordance_index(y_true: np.ndarray, y_pred: np.ndarray) -> float:
                     correct_pairs += true_a < true_b
                 else:
                     correct_pairs += true_a > true_b
-
+    
     assert pairs > 0, 'No pairs for comparision'
-
-    return correct_pairs / pairs
+    
```

Comment:
run `yapf -i file_name`. there are some white spaces.

Reply:
I solved some issues you mentioned above.

Reply:
Is it necessary to make changes in preety_print.py file? if not then don't do it.

Reply:
It's true not required to change the preety_pring.py file. so I upload the original file.